proposals/PROPOSALS.md

```diff
@@ -0,0 +1,156 @@
+5. **Dormant**,  in which case no changes have been made to a HIP in two iterations, it’s marked as dormant and both the PR and issue are closed. Dormant HIPs can be reopened by any person, be it the same or different authors, at which point it will start from the formal evaluation phase.
+
+#### Involving Stakeholders
+
+At this point where a propsal is being reviewed, if there are relevant stakeholders from industry or in the community (e.g. library authors and maintainers who are affected), they should be solicited for comment and brought into the discussion. Remember, proposals may sound good, but it's best to iron out as many details as possible prior to their acceptance. This means that any hesitations stakeholders have with the project should be factored into the proposal's details. 
+
+#### Updating Proposal Statuses
+
+If the author of a particular proposal wants to update the status of their proposal, they should either comment on the issue and tag an HF Technical Track member with a note regarding what status they would like to update the proposal to. It is good to be redundant and raise the question in our other fora as well: Slack and Discourse. However, Github should be the first choice. 
+
+### Formal presentation 
+
+After a proposal is merged, the author will present KPI's and deliverables that define the proposal to the HF Technical track. During the next available Technical Track meeting after acceptance, the author or designated lead (as defined by the proposal) should give a short 5-10min presentation consisting of the following: 
+
+1. **High level summary:** the author will present the high-level abstract for what the proposal intends to accomplish. 
+2. **Timeline:** In what timeline, and should outline possible hurdles and risks that affect the timeline. 
+3. **Resources:** What resources are required in order to take the project from start to completion.
+4.  **Contact:** Who is going to update the HF Technical Track members, and on what schedule.
+
+Once the formal presentation is given, the HF Technical Track members will track the progress over time at our ongoing meetings. This also gives the track members time to introduce themselves and and we can all get to know each other. Currently, HF Technical Track meetings are held weekly on Tuesdays at 11:30am-12:30pm UTC-5 (EST). 
+
+## Structure of the process
+
+The HIP process involves the following parties:
+
+1. The HIP Authors
+2. Relevant stakeholders affected by the change
+3. The HF Technical Track members
+
+### The HIP Authors
+
+Authors are responsible for building consensus within the community and documenting dissenting opinions before the HIP is officially discussed by the HIP HF Technical Track . Their goal is to convince the HF Technical Track that their proposal is useful and addresses pertinent problems in the Haskell ecosystem as well as interactions with already existing features. Authors can change over the life-cycle of the HIP.
```

Comment:
If the decision making body is the Tech Track working group, then I assume HIPs are intended to be primarily technical.   What about proposals that are more social than technical, e.g. "We should run an annual conference" or "HF should affiliate to group X"?

Rather than try to be comprehensive, I suggest restricting this process to stuff within the scope of the Tech Track task force -- and try to write down what that scope is.  (In a word "technical".)